privacy.md

```diff
@@ -0,0 +1,92 @@
+# SponsorLink Privacy Statement
+
+This privacy statement explains the personal data SponsorLink processes, how SponsorLink processes it, 
+and for what purposes.
+
+SponsorLink offers a mechanism for integrators check whether a user is sponsoring a given account. 
+References to SponsorLink products in this statement include SponsorLink backend services, websites, 
+apps, extension and other related software.
+
+This statement applies to the interactions SponsorLink has with you and the SponsorLink products listed below, 
+as well as other SponsorLink products that display this statement.
+
+# Personal data we collect
+
+SponsorLink does not collect any data unless you explicity consent to it. You provide this data directly, 
+by interacting with the SponsorLink GitHub CLI extension and indirectly afterwards by interacting with 
+integrators that use SponsorLink to check whether you are sponsoring a given account. 
+
+The only personal data we collect outside of your local machine is your GitHub user identifier, as 
+part of authenticating with SponsorLink for the purpose of signing a locally-generated manifest file.
+Upon logging in for the first time you will be prompted to Authorize the GH Sponsors CLI with access to 
+your GitHub profile. This authorization (required for manifest signing), alongside with the signature 
+verification of the manifest itself, is sufficient indication to integrators that you have consented 
+to the use of your personal data for the purpose of checking whether you are sponsoring a given account.
+
+Your SponsorLink authentication token is stored locally as an environment variable to avoid having to 
```

Comment:
It might be helpful to enumerate what kind of access this token provides, because I'm sure some folks will be uneasy about an auth token in plaintext in an environment variable. This way they'd understand the worst case scenario if that token was captured by a malicious program that sniffed all the environment variables.

Reply:
Very good point. Updated!